app/assets/scripts/modules/setLegendSize.js

```diff
@@ -0,0 +1,12 @@
+const ChartFactory = require('../../../helpers/chartFactory');
+const _  = require('lodash');
+
+const setLegendSize = (chart, params) => {
+  chart.layout.legend.font.size = params.newValue;
```

Comment:
To be honest: `setLegendSize` is a bit misleading here because you are not setting the width/height of the legend.

Reply:
well we can rename it to `setLegendFontSize`

Reply:
I just like self-describing names, it's still your decision